README.md

```diff
@@ -118,7 +145,7 @@ specification for a full list of information that can be retrieved.
     depstream_list = dep.get_runtime_streams('depstreamname')
     <do_stuff>
 ```
-
+ 
```

Comment:
Could you delete this extra space and force-push?

Reply:
Done